src/DrupalVersionChanger.php

```diff
@@ -558,6 +565,9 @@ private function hasWildcardVersion(string $version): bool
      */
     private function generateCaretVersion(string $version): string
     {
+        if (str_starts_with($version, 'dev-')) {
```

Comment:
```suggestion
        if (substr($version, 0, 4) === 'dev-') {
```

The project requirements specify that this composer plugin needs at least PHP 7.3.0:

![image](https://github.com/user-attachments/assets/a3d35964-680f-4152-b38c-24f8f674f4d9)

However, the `str_starts_with` function is only available in `PHP 8.0+`. To ensure compatibility with `PHP 7.3.0+`, we need to use an alternative method for this check. Failing to do so could result in a `500 error` when the plugin is used in environments running PHP 7.

Thanks.